python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py

```diff
@@ -303,93 +315,96 @@ def test_materialize_checks_no_asset(
-        - AssetSelection.keys(AssetKey(["customers"])).without_checks(),
+        selection=(
+            AssetSelection.keys(AssetKey(["customers"]))
+            - AssetSelection.keys(AssetKey(["customers"])).without_checks()
+        ),
     )
+    assert result.success
     assert len(result.get_asset_materialization_events()) == 0
     assert len(result.get_asset_check_evaluations()) == 2
 
 
-def test_asset_checks_are_logged_from_resource(
-    test_asset_checks_manifest: Dict[str, Any], mocker: MockerFixture, dbt_commands: List[List[str]]
+def test_asset_checks_results(
+    test_asset_checks_manifest: Dict[str, Any], dbt_commands: List[List[str]]
 ):
-    mock_context = mocker.MagicMock()
-    mock_context.assets_def = None
-    mock_context.has_assets_def = True
-
-    dbt = DbtCliResource(project_dir=os.fspath(test_asset_checks_path))
-
-    events = []
-    invocation_id = ""
-    for dbt_command in dbt_commands:
-        dbt_cli_invocation = dbt.cli(
-            dbt_command,
-            manifest=test_asset_checks_manifest,
-            dagster_dbt_translator=dagster_dbt_translator_with_checks,
-            context=mock_context,
-            target_path=Path("target"),
-            raise_on_error=False,
-        )
-
-        events += list(dbt_cli_invocation.stream())
-        invocation_id = dbt_cli_invocation.get_artifact("run_results.json")["metadata"][
-            "invocation_id"
-        ]
-
-    assert (
-        AssetCheckResult(
-            passed=True,
-            asset_key=AssetKey(["customers"]),
-            check_name="unique_customers_customer_id",
-            metadata={
-                "unique_id": (
-                    "test.test_dagster_asset_checks.unique_customers_customer_id.c5af1ff4b1"
-                ),
-                "invocation_id": invocation_id,
-                "status": "pass",
-            },
-        )
-        in events
-    )
-    assert (
-        AssetCheckResult(
-            passed=True,
-            asset_key=AssetKey(["customers"]),
-            check_name="not_null_customers_customer_id",
-            metadata={
-                "unique_id": (
-                    "test.test_dagster_asset_checks.not_null_customers_customer_id.5c9bf9911d"
-                ),
-                "invocation_id": invocation_id,
-                "status": "pass",
-            },
-        )
-        in events
-    )
-    assert AssetCheckResult(
-        passed=False,
-        asset_key=AssetKey(["fail_tests_model"]),
-        check_name="unique_fail_tests_model_id",
-        severity=AssetCheckSeverity.WARN,
-        metadata={
-            "unique_id": ("test.test_dagster_asset_checks.unique_fail_tests_model_id.1619308eb1"),
-            "invocation_id": invocation_id,
-            "status": "warn",
-        },
+    @dbt_assets(
+        manifest=test_asset_checks_manifest,
+        dagster_dbt_translator=dagster_dbt_translator_with_checks,
     )
-    assert AssetCheckResult(
```

Comment:
These tests weren't actually doing anything, since there was no `in events` call

Reply:
:o